.ado/pipelines/templates/stages-full-release.yaml

```diff
@@ -168,7 +168,7 @@ stages:
                                     -var=queued_by="$(Build.QueuedBy)"
                                     -var=contact_email="$(contactEmail)"
                                     -var=''stamps=$(stampLocations)''
-                                    -var=kubernetes_version="$(kubernetesVersion)"
+                                    -var=aks_kubernetes_version="$(kubernetesVersion)"
```

Comment:
aks_kubernetes is somewhat redundant, dont you think? how about aks_version?

Reply:
I did change this for consistency. All the other "aks" variables had "aks_" as a prefix. And - it's not the aks_version. It's the kubernetes_version used for this specific AKS (node pool - though we're not using different vers for cluster and node pools).

Reply:
![image](https://user-images.githubusercontent.com/17407022/166675412-57a869c1-7e0d-4f97-a575-39a072f3bbeb.png)

Reply:
what other aks version can there be, then? If you dont want to call it aks_version?

Reply:
At some point maybe the AKS Release version:

![image](https://user-images.githubusercontent.com/17407022/166675707-e6bd3fca-b191-445b-9d7c-b2f978f8024b.png)

Reply:
Why should this parameter that configures an AKS setting not start with the AKS prefix like the others?

Reply:
I'm ok to keep it like this if you think there is potential future use. its just that Azure Kubernetes Service Kubernetes Version felt a bit redundant to me

Reply:
I see your point, but it's not the "aks version" but still a parameter for "aks" so I'd like to have it consistent using a common prefix "aks_" for all aks variables.